Tvl.VisualStudio.InheritanceMargin.CSharp/CSharpInheritanceAnalyzer.cs

```diff
@@ -275,7 +275,7 @@ protected override void ReParseImpl()
                         ISet<ITypeSymbol> derivedTypes = new HashSet<ITypeSymbol>();
                         derivedTypes.UnionWith(FindDerivedClassesAsync.Value(typeSymbol, solution, null, CancellationToken.None).Result);
                         derivedTypes.UnionWith(FindDerivedInterfacesAsync.Value(typeSymbol, solution, null, CancellationToken.None).Result);
-                        derivedTypes.UnionWith(FindImplementingTypesAsync.Value(typeSymbol, solution, null, CancellationToken.None).Result);
+                        derivedTypes.UnionWith(FindImplementingTypesAsync.Value(typeSymbol, solution, ImmutableHashSet.CreateRange(solution.Projects), CancellationToken.None).Result);
```

Comment:
:exclamation: This change was unintentional. I was experimenting with "Metadata as Source" projects to see if I could make e.g. implementations of `IEquatable<T>` show up.